environment.yml

```diff
@@ -11,3 +11,4 @@ dependencies:
 - pyarrow
 - matplotlib
 - loguru
+- joblib
```

Comment:
Also update any other places where installation is needed (pyproject,toml, CI devtools/conda-envs/test_env.yaml, RTD, later the cf feedstock)